### DISCUSSION TITLE
Changes in nodes

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#6980] [Work In Progress - GUI] [answered]
Today I see my nodes have no backdrop on develop. Is this expected?

![image](https://github.com/enso-org/enso/assets/3919101/c3d8d08e-9de5-4708-9c1c-b2d71e91da49)

**Comment by wdanilo** [comment, marked as answer]
Yes! The new design does not have shadows with nodes and Kaz recently optimized them to use our super-box to lower render draw count :) 

<img width="839" alt="image" src="https://github.com/enso-org/enso/assets/1623053/fd27fd4d-97d5-47e3-b828-ae0bcf74ca36">
